pkgs/development/tools/build-managers/bazel/buildtools/default.nix

```diff
@@ -1,16 +1,16 @@
-{ stdenv, buildGoPackage, fetchFromGitHub }:
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
```

Comment:
`fetchhg, fetchbzr, fetchsvn` don't seem to be used, were they added by `go2nix`?

Reply:
Yes, they were added by go2nix.